lib/chef/provider/git.rb

```diff
@@ -105,7 +105,7 @@ def action_sync
       end
 
       def git_minor_version
-        @git_minor_version ||= Gem::Version.new(shell_out!("git --version", run_options).stdout.split.last)
+        @git_minor_version ||= Gem::Version.new((run_git_command ["--version"]).stdout.split.last)
```

Comment:
we typically leave parens around method calls to make it clearer (like `#new()` and `#git()`).  i think there's a few exceptions where its typical ruby style to leave them off and its unambiguous...

so:

```
@git_minor_version ||= Gem::Version.new( git("--version").stdout.split.last )
 ```

i like leaving the space around the slightly complicated argument so that visually its clear and the expression can 'breathe' a little more--which is why the cop that removes those spaces is disabled.